docs/internals/contributing/writing-code/coding-style.txt

```diff
@@ -67,6 +67,11 @@ Python style
   message. Use :meth:`~unittest.TestCase.assertRaisesRegex` only if you need
   regular expression matching.
 
+* In tests, use :meth:`~django.test.SimpleTestCase.assertWarnsMessage` instead
```

Comment:
I'd combine this with the prefix note, "In tests, use assertRaisesMessage/assertWarnsMessage instead of assertRaises/assertWarns so ....